persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -72,6 +79,16 @@ class HmdaFileValidator(submissionId: String) extends HmdaPersistentActor with L
       val validationStarted = ValidationStarted(submissionId)
       publishEvent(validationStarted)
       events(parserPersistenceId)
+        .filter(x => x.isInstanceOf[TsParsed])
```

Comment:
There's unfortunately no `find` method for a `Flow`.  I think `take(1)` could work, but I'm not sure if we can guarantee the order of the events.